ebml/EbmlElement.h

```diff
@@ -220,6 +307,7 @@ class EBML_DLL_API EbmlCallbacks {
         inline EbmlElement & NewElement() const { return Create(); }
         /// is infinite/unknown size allowed
         inline bool CanHaveInfiniteSize() const { return CanInfinite; }
+        virtual const bool HasDefault() const { return false; }
```

Comment:
It seems this is not possible for static instances with DLL exporting.

@mbunkus do you still use libebml/libmatroska as DLL on Windows ?

Reply:
I don't. For Windows I only link statically. On other OS I can link dynamically (which is easier to do there), or rather, on those I really don't care how it's linked. For Windows I haven't tried using DLLs for libEBML/libMatroska in a very, very long time.

Reply:
OK, I'm thinking about removing support for Windows. If you look at all the [warnings](https://github.com/Matroska-Org/libebml/actions/runs/7338935322/job/19982276669), it's clear we're not doing it right. For example we're supposed to be hiding or making public some static: https://learn.microsoft.com/en-us/cpp/cpp/using-dllimport-and-dllexport-in-cpp-classes. To do it right we should have our internal class and the version with just the bits that we want to make available to other classes. Users of our exported classes might not be using the same STL: https://stackoverflow.com/a/11189727. And it seems we can't export CRTError properly for the same reason.

We are not using it in VLC except if it's build like that on the Linux distro.